recipe/build.sh

```diff
@@ -23,6 +43,54 @@ if [[ $target_platform == osx-64 ]]; then
     export CXXFLAGS="${CXXFLAGS:-} -D_LIBCPP_DISABLE_AVAILABILITY=1"
 fi
 
+if [[ $target_platform == osx-arm64 ]] && [[ $CONDA_BUILD_CROSS_COMPILATION == 1 ]]; then
+    # Build all intermediate codegen binaries for the build platform
+    # xref: https://cmake.org/pipermail/cmake/2013-January/053252.html
+    env -u SDKROOT -u CONDA_BUILD_SYSROOT -u CMAKE_PREFIX_PATH \
+        -u CXXFLAGS -u CPPFLAGS -u CFLAGS -u LDFLAGS \
```

Comment:
You can do something like https://github.com/conda-forge/gdk-pixbuf-feedstock/blob/b8ad56912306789de031f7f12db041630765d3fa/recipe/build.sh#L44-L56 which makes it more general than osx-arm64.

Reply:
I'm running the two builds only in the case of osx-arm64. At the moment, I rely on the qemu stuff for ppc64le. In case we add another cross build, I'll convert it to a bit more generic step.